server/src/schema.js

```diff
@@ -39,7 +38,7 @@ export const Schema = [`
 
   input AddUserToGroupInput {
     groupId: Int!
-    userId: Int!
+    userId: Int
```

Comment:
why would this become optional?

Reply:
Because the client doesn't know its client ID, so we shouldn't make it required to make this call. the server can do the lookup.

Reply:
good call - we should remove it entirely.

Reply:
we can make it optional later if we want to support adding users other than ourselves.